type-definitions/tests/immutable-flow.js

```diff
@@ -268,6 +272,41 @@ stringToNumber = Map({'a': 1}).mapKeys((key, value, iter) => 1)
 
 anyMap = Map({'a': 1}).flatten()
 
+/* OrderedMap */
+
+orderedStringToNumber = Map({'a': 1}).toOrderedMap()
+orderedStringToNumber = OrderedMap({'a': 1})
+orderedStringToNumber = OrderedMap(Map({'a': 1}))
+orderedStringToNumber = OrderedMap()
+orderedStringToNumber = OrderedMap().set('b', 2)
+orderedStringToNumber = OrderedMap({'a': 1}).delete('a')
+orderedStringToNumber = OrderedMap({'a': 1}).remove('a')
+orderedStringToNumber = OrderedMap({'a': 1}).clear()
+orderedStringToNumber = OrderedMap({'a': 1}).update(() => OrderedMap({'b': 1}))
+orderedStringToNumber = OrderedMap({'a': 1}).update('a', value => value + 1)
+orderedStringToNumber = OrderedMap({'a': 1}).update('a', 0, value => value + 1)
+orderedStringToNumber = OrderedMap({'a': 1}).merge({'b': 2})
+orderedStringToNumber = OrderedMap({'a': 1}).mergeWith((prev, next) => next, [1])
+orderedStringToNumber = OrderedMap({'a': 1}).mergeDeep({'a': 2})
+orderedStringToNumber = OrderedMap({'a': 1}).mergeDeepWith((prev, next) => next, [1])
+orderedStringToNumber = OrderedMap({'a': 1}).setIn([], 3)
+orderedStringToNumber = OrderedMap({'a': 1}).deleteIn([])
+orderedStringToNumber = OrderedMap({'a': 1}).removeIn([])
+orderedStringToNumber = OrderedMap({'a': 1}).updateIn([], val => val + 1)
+orderedStringToNumber = OrderedMap({'a': 1}).updateIn([], 0, val => val + 1)
+orderedStringToNumber = OrderedMap({'a': 1}).mergeIn([], {'b': 2})
+orderedStringToNumber = OrderedMap({'a': 1}).mergeDeepIn([], {'b': 2})
+orderedStringToNumber = OrderedMap({'a': 1}).withMutations(mutable => mutable.set('b', 2))
+orderedStringToNumber = OrderedMap({'a': 1}).asMutable();
+orderedStringToNumber = OrderedMap({'a': 1}).asImmutable();
+orderedStringToNumber = OrderedMap({'a': 1}).map((v) => v + 1)
+orderedStringToNumber = OrderedMap({'a': 1}).flatMap((v, k) => (OrderedMap({ [k]: v + 1 })))
+orderedStringToNumber = OrderedMap({'a': 1}).flip().flip()
+orderedStringToNumber = OrderedMap({'a': 1}).mapKeys(x => x)
+orderedStringToNumber = OrderedMap({'a': 1}).flatten()
+orderedStringToNumber = OrderedMap({'a': 1}).flatten(1)
+orderedStringToNumber = OrderedMap({'a': 1}).flatten(true)
```

Comment:
Thanks for the tests! It would be great if there were tests for expected errors, too. You can do it with `// $ExpectError`. See [here](https://github.com/facebook/immutable-js/pull/1027/files#diff-167148588528b5fa451dc535e9c29037R399`).